benchmarking/smartseq2/RunBenchmarkingAnalysis.json

```diff
@@ -0,0 +1,13 @@
+{
+  "RunBenchmarkingAnalysis.scripts_dir": "gs://broad-dsde-mint-dev-teststorage/pipeline_testing_scripts",
```

Comment:
These files should be in the public mint bucket, I think. @dshiga @TimothyTickle Do you guys agree?

Reply:
we don't need this any more. all the scripts are in docker now. I will remove this line from json.